client/src/templates/Challenges/classic/classic.css

```diff
@@ -215,6 +219,10 @@
   width: 100%;
 }
 
+#mobile-layout .monaco-editor-tabs:empty {
+  padding: 0;
+}
```

Comment:
If it is empty, it does not have to be rendered.

Reply:
I agree with you.

Therefore, with using `isRenderChallengeFiles`, the logic for displaying `.monaco-editor-tabs` will be added and the redundant CSS code will be removed.